packages/vaadin-upload/src/vaadin-upload-file.js

```diff
@@ -28,7 +28,7 @@ import './vaadin-upload-icons.js';
  * `commands` | Container for file command icons
  * `start-button` | Start file upload button
  * `retry-button` | Retry file upload button
- * `clear-button` | Clear file button
+ * `remove-button` | Remove file button
```

Comment:
Let's exclude this change from this PR, and then create a separate PR for it. This way it will be better emphasised in release notes, and easier to keep in mind when creating migration guide.

Reply:
Does it mean that I should temporarily set the aria-label attribute on the Clear/Remove button to "Clear"?

Reply:
Yes, let's set it to `Clear` for now and then rename in the following PR.